doc/classes/SurfaceTool.xml

```diff
@@ -37,6 +37,7 @@
 			</argument>
 			<description>
 				Specifies a [Color] for the next vertex to use.
+				[b]Note:[/b] The material must have [member BaseMaterial3D.vertex_color_use_as_albedo] enabled for the vertex color to be visible. Also, if no lighting is used in the scene, set the material's [member BaseMaterial3D.shading_mode] property to [constant BaseMaterial3D.SHADING_MODE_UNSHADED].
```

Comment:
I'm not sure mentioning ``UNSHADED`` is a good idea here. It has nothing to do with vertex colors. It just so happened that in that particular issue the user had an odd setup.

Reply:
Bump.